full/Dockerfile

```
@@ -275,6 +276,24 @@ ENV PATH=$PATH:$HOME/.cargo/bin
 
 RUN bash -lc "cargo install cargo-watch cargo-edit cargo-tree"
 
+### Docker ###
+LABEL dazzle/layer=tool-docker
+LABEL dazzle/test=tests/tool-docker.yaml
+USER root
+# https://docs.docker.com/engine/install/ubuntu/
+RUN curl -o /var/lib/apt/dazzle-marks/docker.gpg -fsSL https://download.docker.com/linux/ubuntu/gpg && apt-key add /var/lib/apt/dazzle-marks/docker.gpg \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" \
+    && apt-get update \
+    && apt-get install -y docker-ce docker-ce-cli containerd.io slirp4netns \
```

I think we should install `docker-compose` as well -- that's one of the top-requested Docker use cases (more than `dive` 😇) and would be a really nice surprise for users to have it out-of-the-box. 🎁

```suggestion
    && apt-get install -y docker-ce docker-ce-cli containerd.io slirp4netns docker-compose \
```